docs/content/getting-started/project-file-reference.mdx

```diff
@@ -0,0 +1,438 @@
+      </td>
+    </tr>
+
+  </tbody>
+</table>
+        </td>
+    </tr>
+    <tr>
+      <td>my_dagster_project_tests/</td>
+      <td>
+        A Python package that contains tests for <code>my_dagster_project</code>
+        .
+      </td>
+    </tr>
+    <tr>
+      <td>README.md</td>
+      <td>A description and starter guide for the Dagster project.</td>
+    </tr>
+    <tr>
+      <td>pyproject.toml</td>
+      <td>
+        A file that specifies package core metadata in a static, tool-agnostic
+        way.
+        <br />
+        <br />
+        This file includes a <code>tool.dagster</code> section which references the
+        Python package with your Dagster definitions defined and discoverable at
+        the top level. This allows you to use the
+        <code>dagster dev</code> command to load your Dagster code without any
+        parameters. Refer to the{" "}
+        <a href="/concepts/code-locations">Code locations documentation</a> to
+        learn more.
+        <br />
+        <br /> <strong>Note:</strong> <code>pyproject.toml</code> was introduced
+        in{" "}
+        <a href="https://peps.python.org/pep-0518/https://peps.python.org/pep-0518/">
+          PEP-518
+        </a>{" "}
+        and meant to replace <code>setup.py</code>, but we may still include a{" "}
+        <code>setup.py</code> for compatibility with tools that do not use this
+        spec.
+      </td>
+    </tr>
+    <tr>
+      <td>setup.py</td>
+      <td>
+        A build script with Python package dependencies for your new project as
+        a package.
+        <br /><br />
+        <strong>Note</strong>: If using Dagster Cloud, <code>dagster-cloud</code> as a dependency.
+      </td>
+    </tr>
+    <tr>
+      <td>setup.cfg</td>
+      <td>
+        An ini file that contains option defaults for <code>setup.py</code>{" "}
+        commands.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+---
+
+## Configuration files
+
+<Note>
+  <strong>Not sure where to put these files?</strong> Unless otherwise noted,
+  all configuration files should be placed in the root of your Dagster project
+  directory.
+</Note>
+
+Depending on your use case or if you're using Dagster Cloud, you may also need to add additional configuration files to your project:
+
+<table
+  className="table"
+  style={{
+    width: "100%",
+  }}
+>
+  <thead>
+    <tr>
+      <th
+        style={{
+          width: "20%",
+        }}
+      >
+        File/Directory
+      </th>
+      <th>Description</th>
+      <th
+        style={{
+          width: "10%",
+        }}
+      >
+        OSS
+      </th>
+      <th
+        style={{
+          width: "10%",
+        }}
+      >
+        Cloud
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>dagster.yaml</td>
+      <td>
+        Configures your Dagster instance, including defining storage locations,
```

Comment:
Most hybrid agents are not configured directly by a dagster.yaml file. Instead, the k8s agent is configured through a helm chart that, in turn, creates a dagster.yaml file (via a ConfigMap) and ECS is configured through a CloudFormation template that generated a dagster.yaml file as part of the agent task definition.

Reply:
Is your comment referring to "Configures your Dagster instance", or the bit about "Using this file to customize a Hybrid agent"?

Reply:
I meant that "Using this file" feels incorrect, since most people do not literally use dagster.yaml. Perhaps we can just add a note that this file is often configured indirectly through helm or cloud formation?